wordpress-mobile/WordPress-iOS
# Add gravatar to the Me button

#### Download the gravatar and add it to the new `Me` button in `My Site`. If gravatar is unavailable, fall back to the default `Me` icon (circled user).

Ref #13319

##### Change log
- Update BlogDetailsViewController+MeBarButton.swift, add UIBarButtonItem extension that downloads the gravatar and sets it to the custom view of the button
- Update CircularImageView.swift, add an optional tap animation to animate the tap on the image

##### Notes

1. The gravatar size is set to the default `GridIcon` size (24 x 24). TBD with @osullivanchris if this needs to be changed.
2. Known issue: when rotating to landscape, the top bars are misaligned in the split view
3. This PR does not remove the original `Me` scene in the tab bar (yet)
4. This PR does not include unit tests


To test:
1. Build/run and login with an account that has a gravatar image
2. Choose a site of your preference and go to `My Site`
3. Verify that the `Me` button contains the gravatar.
4. Tap the `Me` button and verify that the `Me` scene is presented modally and works as expected.
5. Repeat steps 1 and 2 with an account that has no gravatar, and verify that the default image (circled user) is displayed on the `Me` button in `My Site`.


PR submission checklist:
